lib/ansible/module_utils/basic.py

```diff
@@ -2679,6 +2680,10 @@ def _clean_args(self, args):
 
         return self._clean
 
+    def _restore_signal_handlers(self):
+        if sys.platform != 'win32':
+            signal.signal(signal.SIGPIPE, signal.SIG_DFL)
+
```

Comment:
You should probably add a comment here in the code about why this is being done, so that someone doesn't have to go spelunking back through git and the PR to find the reasoning.